spec.md

```diff
@@ -1365,116 +1365,18 @@ If the plugin is unable to complete the NodeGetCapabilities call successfully, i
-  * Plugin Packages SHOULD use a well-documented container image format (e.g., Docker, OCI).
-  * The chosen package image format MAY expose configurable Plugin properties as environment variables, unless otherwise indicated in the section below.
-    Variables so exposed SHOULD be assigned default values in the image manifest.
-  * A Plugin Supervisor MAY programmatically evaluate or otherwise scan a Plugin Package’s image manifest in order to discover configurable environment variables.
-  * A Plugin SHALL NOT assume that an operator or Plugin Supervisor will scan an image manifest for environment variables.
-
-#### Environment Variables
-
-* Variables defined by this specification SHALL be identifiable by their `CSI_` name prefix.
-* Configuration properties not defined by the CSI specification SHALL NOT use the same `CSI_` name prefix; this prefix is reserved for common configuration properties defined by the CSI specification.
-* The Plugin Supervisor SHOULD supply all recommended CSI environment variables to a Plugin.
-* The Plugin Supervisor SHALL supply all required CSI environment variables to a Plugin.
-
-##### `CSI_ENDPOINT`
-
-Network endpoint at which a Plugin SHALL host CSI RPC services. The general format is:
-
-    {scheme}://{authority}{endpoint}
-
-The following address types SHALL be supported by Plugins:
-
-    unix:///path/to/unix/socket.sock
-
-Note: All UNIX endpoints SHALL end with `.sock`. See [gRPC Name Resolution](https://github.com/grpc/grpc/blob/master/doc/naming.md).  
-
-This variable is REQUIRED.
-
-#### Operational Recommendations
-
-The Plugin Supervisor expects that a Plugin SHALL act as a long-running service vs. an on-demand, CLI-driven process.
-
-Supervised plugins MAY be isolated and/or resource-bounded.
-
-##### Logging
-
-* Plugins SHOULD generate log messages to ONLY standard output and/or standard error.
-  * In this case the Plugin Supervisor SHALL assume responsibility for all log lifecycle management.
-* Plugin implementations that deviate from the above recommendation SHALL clearly and unambiguously document the following:
-  * Logging configuration flags and/or variables, including working sample configurations.
-  * Default log destination(s) (where do the logs go if no configuration is specified?)
-  * Log lifecycle management ownership and related guidance (size limits, rate limits, rolling, archiving, expunging, etc.) applicable to the logging mechanism embedded within the Plugin.
-* Plugins SHOULD NOT write potentially sensitive data to logs (e.g. `Credentials`).
-
-##### Available Services
-
-* Plugin Packages MAY support all or a subset of CSI services; service combinations MAY be configurable at runtime by the Plugin Supervisor.
-* Misconfigured plugin software SHOULD fail-fast with an OS-appropriate error code.
-
-##### Linux Capabilities
-
-* Plugin Supervisor SHALL guarantee that plugins will have `CAP_SYS_ADMIN` capability on Linux when running on Nodes.
```

Comment:
we need this so that plugins can execute mount operations. i don't think it's wise to remove from the spec.

Reply:
Agree it's ok to remove, this is a packaging concern and not related to how the CO and the SP can interact.